docs/walkthrough_collaborate.md

```diff
@@ -31,23 +31,20 @@ Only the owner of a team is allowed to add or remove members or to transfer owne
 
 ## Invite a user
 
-In eScriptorium, some users can have the right to invite new users to an instance. The invitation rights are not activated for everyone.
+In eScriptorium, some users can have the right to invite new users. The invitation rights are not activated for everyone.
 
 You can find more information about inviting new users in the [user-related features walkthrough](walkthrough_users.md#invite) section.
 
 ## Share a project or a document
 
-You can share projects and documents to others users of the same eScriptorium instance.
+You can share projects and documents with others users of the same eScriptorium instance.
 
 ### Share a project
 
-You can share an opened project you own by clicking on the "Share this project" button, located in the project dashboard at {base_url}/project/{project_slug}/documents/. **Only the owner of a project can share it.** It will then open a pop-up box where you have two options:
```

Comment:
I wasn't sure why you put "an opened project" here?

Reply:
It's just because the button is accessible from an opened project! :)

Reply:
what do you mean by "opened project"?